### PR TITLE
Make feature_requests a YAML

### PR DESCRIPTION
**Pull request by Mbrick2**
I don't know how I would test this.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: is-related
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description:A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (webdev03) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:14]: there needs to be a space in between colon and text

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature Request]: <title>"
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```
Comment (webdev03) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:9]: this isn't a bug report

**Review by webdev03 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 1 above)
Comment (Mbrick2) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: [Resolved](https://github.com/JaydenDev/Catalyst/pull/180/commits/42d0a058965bdabd04382cf88cdfc5ca7ff12a22)

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 2 above)
Comment (Mbrick2) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: [Fixed](https://github.com/JaydenDev/Catalyst/pull/180/commits/42d0a058965bdabd04382cf88cdfc5ca7ff12a22)

**Comment by Mbrick2** [comment]
@JaydenDev Ready to merge?

**Pull request merged**